### DISCUSSION TITLE
Tramming wizard and mesh leveling does not show height numbers

### DISCUSSION THREAD

**Discussion by loadingdread** [mriscoc/Ender3V2S1#1118] [Q&A]
Apologies if I am missing something obvious. I have recently installed  Ender3S1-F4-UBL-20230904 on my ender 3 and generally its working well. However, in the tramming wizard and mesh leveling screens all I see are coloured squares, with no numbers appearing to indicate height. Is there a setting I am missing?

**Comment by mriscoc** [comment]
TJC

**Comment by loadingdread** [comment]
For anyone else would couldnt find the solution, its here https://github.com/mriscoc/Ender3V2S1/issues/542#issuecomment-1345621590

Thanks mriscoc :)
